XML/TestCases/PerformanceTests.xml

```diff
@@ -184,8 +184,8 @@
         <Priority>2</Priority>
     </test>
     <test>
-        <testName>PERF-NETWORK-TCP-LATENCY-MULTICONNECTION-Synthetic</testName>
-        <testScript>PERF-NETWORK-TCP-LATENCY-MULTICONNECTION-LAGSCOPE.ps1</testScript>
+        <testName>PERF-NETWORK-TCP-LATENCY-Synthetic</testName>
```

Comment:
May be PERF-NETWORK-TCP-LATENCY-2CLIENT-Synthetic & PERF-NETWORK-TCP-LATENCY-2CLIENT-SRIOV?

Reply:
This is not my change. Will revert this file change.